### DISCUSSION TITLE
cfclient_deck tab

### DISCUSSION THREAD

**Discussion by MINJEEgi** [bitcraze/discussions#601] [Q&A]
Hi:)
Should the Deck Expansion menu or Deck Tab be visible in cfclient? Are there any specific settings required when using the Loco Positioning System in cfclient? I'm also curious about the Deck Tab.
Even when we connect the Crazyflie, we can't see the Deck Tab. It doesn't show up.

**Comment by krichardsson** [comment]
I'm not sure I fully understand the question. You can find the tabs that are available in the 'View/tabs` menu. This menu is always the same, regardless of the decks that are mounted on the Crazyflie. This means that you can activate a tab that is related to a specific deck, for instance the `Lighthouse positioning` tab, also without a lighthouse deck. If the deck is not available the contents of the tab will simply be disabled.

> I'm also curious about the Deck Tab.

There is no `Deck` tab as far as I know. Is it mentioned somewhere in the documentation?
